tools/tpm2_pkcs11/db.py

```diff
@@ -240,7 +242,8 @@ def create(self):
                 id INTEGER PRIMARY KEY,
                 hierarchy TEXT NOT NULL,
                 handle BLOB NOT NULL,
-                objauth TEXT NOT NULL
+                objauth TEXT NOT NULL,
+                pobjauthpolicy BLOB
```

Comment:
primary objects have a policy hash? Why?

Reply:
So that the only operations allowed are:
1. Import 
2. Load externally created objects
3. Create objects with the intended policy hash. This means that objects explicitly created with tpm2-pkcs11 are guaranteed to have a policy that accommodates the new auth model.